docs/integrations/creating-nodes/build/reference/ui-elements.md

```diff
@@ -564,4 +564,95 @@ The HTML editor allows users to create HTML templates in their workflows. The ed
 
 Refer to [`Html.node.ts`](https://github.com/n8n-io/n8n/blob/master/packages/nodes-base/nodes/Html/Html.node.ts){:target=_blank .external-link} for a live example.
 
+## Resource mapper
 
+If your node performs insert, update, or upsert operations, you need to send data from the node in a format supported by the service you're integrating with. A common pattern is to use a Set node before the node that sends data, to convert the data to match the schema of the service you're connecting to. The resource mapper UI component provides a way to get data into the required format directly within the node, rather than using a Set node.
```

Comment:
I think we should mention that _getting the data in the right format_ means that RMC will validate input against the schema and also cast it into expected type.